applications/openshift/controller/controller_bind_address/rule.yml

```diff
@@ -34,11 +34,15 @@ ocil_clause: |-
     <tt>bind-address</tt> is not configured to a secure IP address
 
 ocil: |-
-    To verify that <tt>bind-address</tt> is configured correctly,
+    To verify that <tt>secure-port</tt> is configured correctly,
     run the following command:
-    <pre>$ oc get configmaps config -n openshift-kube-controller-manager -ojson | jq -r '.data["config.yaml"]' | jq -r '.extendedArguments["bind-address"]'</pre>
-    Verify that the <tt>bind-address</tt> argument is set to a secure
-    IP address.
+    <pre>$ oc get configmaps config -n openshift-kube-controller-manager -ojson |   jq -r '.data["config.yaml"]' | jq '.extendedArguments["secure-port"][]'</pre>
+    Verify that it's using an appropriate port (the value is not <pre>0</pre>).
+
+    To verify that <tt>port</tt> is configured correctly,
+    run the following command:
+    <pre>$ oc get configmaps config -n openshift-kube-controller-manager -ojson |   jq -r '.data["config.yaml"]' | jq '.extendedArguments["port"][]'</pre>
```

Comment:
Rules that check multiple different configurations need to be broken out into separate rules. This makes the content more flexible, is a requirement for CCEs, and makes it easier for our end users to understand.

Reply:
Done